supabase/migrations/20230315090547_remote_commit.sql

```diff
@@ -0,0 +1,1124 @@
+--
```

Comment:
I'd suggest splitting this migration file into 3 files:
- `init_database` with Supabase's default configuration
- `add_device_count_tables` with our 3 tables
- `add_access_points_triggers` with our triggers and functions

Naming can probably be better, but you get the point :)